packages/odyssey-react/src/components/Box/_box-text.scss

```diff
@@ -38,6 +42,10 @@
   font-size: var(--FontSizeBase);
 }
 
+.fontSizeCaption {
+  font-size: var(--FontSizeCaption);
+}
+
```

Comment:
see my comment here:

can we avoid adding more API surface area to Box by using composition of <Text> which already has these styles?

Reply:
I had removed this from the box component props, but didn't remove the style. good catch, removing now.